src/quo2/components/buttons/slide_button/component_spec.cljs

```diff
@@ -82,17 +81,11 @@
       (h/has-style track-mock {:opacity constants/disable-opacity})))
 
   (h/test "render the small button"
-    (h/render [slide-button/view (assoc default-props :size :small)])
+    (h/render [slide-button/view (assoc default-props :size :size/s-40)])
```

Comment:
the decision was agreed upon within internal communications channels. It should be added to the docs. I guess we wanted to use it once or twice to make sure we have it right before adding it to the docs. I'll see if someone can add it (if not I will 👍 )
btw, Ulises made a good point and I was curious of the difference, why not `:size-40` instead of `:size/s-40`? does the `/` really give us some benefit?